src/python/pants/backend/debian/target_types.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.core.goals.package import OutputPathField
+from pants.engine.target import (
+    COMMON_TARGET_FIELDS,
+    DictStringToStringField,
+    SpecialCasedDependencies,
+    StringField,
+    Target,
+)
+from pants.util.docutil import doc_url
+
+
+class DebianControlFile(StringField):
+    alias = "control"
+    help = "Path to a Debian control file for the package to be produced."
+
+
+class DebianSymlinks(DictStringToStringField):
+    alias = "symlinks"
+    help = "Symlinks to create for each target being packaged."
```

Comment:
I'm not following how this field will be used. Perhaps it could be added in a followup once we get the initial MVP working?

Reply:
The idea is that given this Debian package structure

```
├── example-pex
│   ├── DEBIAN
│   │   └── control
│   ├── opt
│   │   └── company
│   │       └── example
│   │           └── example-linux-cp3.6.pex
│   └── usr
│       └── bin
│           └── do-work -> /opt/company/example/example-linux-cp3.6.pex
```

user can specify the name of a symlink to be created which will point to the PEX file. In the example above, without a symlink user would have to type `/opt/company/example/example-linux-cp3.6.pex` to run the PEX file or `example-linux-cp3.6.pex` if `/opt/company/example/` happens to be on the `$PATH`. With the symlink created, user will be able to type `do-work` (because `/usr/bin` happens to be on the `$PATH`) without referring to the actual `.pex` file.

Reply:
Got it, thanks! I think an example would help users a lot. Something like:

> \n\nFor example, you could set `symlinks={'blah': 'foo'}`.

(but using better names hah)